ci/prow/boskos/create_projects.sh

```diff
@@ -21,13 +21,56 @@ readonly NUMBER=${2:?"Second argument is the number of new projects."}
 readonly BILLING_ACCOUNT=${3:?"Third argument must be the billing account."}
 readonly OUTPUT_FILE=${4:?"Fourth argument should be a file name all project names will be appended to in a resources.yaml format."}
 
+readonly CUSTOM_ROLE_NAME="KnativeIntegrationTestsRunner"
+readonly CUSTOM_ROLE_FILE="custom_role.yaml"
+
+readonly PROJECT_OWNERS=("prime-engprod-sea@google.com")
+readonly PROJECT_GROUPS=("knative-productivity-admins@googlegroups.com")
+readonly PROJECT_SAS=(
+    "knative-tests@appspot.gserviceaccount.com"
+    "prow-job@knative-tests.iam.gserviceaccount.com"
+    "prow-job@knative-nightly.iam.gserviceaccount.com"
+    "prow-job@knative-releases.iam.gserviceaccount.com")
+readonly PROJECT_APIS=(
+    "cloudresourcemanager.googleapis.com"
+    "compute.googleapis.com"
+    "container.googleapis.com")
+
 for (( i=0; i<${NUMBER}; i++ )); do
   PROJECT="knative-boskos-$(( i + ${FIRST} ))"
   # This Folder ID is google.com/google-default
   # If this needs to be changed for any reason, GCP project settings must be updated.
   # Details are available in Google's internal issue 137963841.
   gcloud projects create ${PROJECT} --folder=396521612403
   gcloud beta billing projects link ${PROJECT} --billing-account=${BILLING_ACCOUNT}
-  "$(dirname $0)/set_permissions.sh" ${PROJECT}
-  echo "  - ${PROJECT}" >> ${OUTPUT_FILE}
+
+  # Set permissions for users on this new project
+  # Add an owner to the PROJECT
+  for owner in ${PROJECT_OWNERS[@]}; do
```

Comment:
I removed `set_permissions.sh` and moved all code here, since we only run it once when creating new projects.